src/custom/hooks/useUSDCPrice/index.ts

```diff
@@ -76,38 +82,45 @@ export default function useUSDCPrice(currency?: Currency) {
 
   useEffect(() => {
     const isSupportedChain = supportedChainId(chainId)
-    if (!isSupportedChain || !currency || !amountOut || !stablecoin) return
+    if (!isQuoteLoading || !isSupportedChain || !sellTokenAddress || !sellTokenDecimals) return
 
-    const params = {
-      baseToken: stablecoin.address,
-      quoteToken: currency.wrapped.address,
+    const baseAmount = STABLECOIN_AMOUNT_OUT[isSupportedChain]
```

Comment:
Not from this PR but `isSupportedChain` is not a flag, is it?
Would you mind updating the name?